src/test/scala/mesosphere/marathon/core/task/tracker/impl/TaskOpProcessorImplTest.scala

```diff
@@ -22,45 +37,6 @@ class TaskOpProcessorImplTest
   // ignored by the TaskOpProcessorImpl
   val deadline = Timestamp.zero
 
-  test("process noop") {
```

Comment:
There is no `TaskStateOp.Noop` or adequate. The case when NoChange is needed is handled in _process statusUpdate with NoChange_